Resources/Prototypes/Loadouts/role_loadouts.yml

```diff
@@ -53,8 +53,10 @@
 - type: roleLoadout
   id: JobServiceWorker
   groups:
+  - ServiceWorkerHead # DeltaV
   - Scarfs # DeltaV
-  - BartenderJumpsuit
+  #- BartenderJumpsuit
+  - ServiceWorkerJumpsuit # DeltaV, replaces above
```

Comment:
probably better to just say was bartender and remove the commented out one

Reply:
Got it, thanks!